db/discord_db/base.py

```diff
@@ -0,0 +1,4 @@
+from sqlalchemy.orm import DeclarativeBase
```

Comment:
Move all modules to a module.py file instead of a file for each module.
File for each class relevant only for Python functionality.